src/packages/server/index.js

```diff
@@ -26,9 +27,11 @@ class Server {
 
   router: Server$opts.router;
 
+  cors: Server$opts.server.cors;
+
   instance: HTTPServer;
 
-  constructor({ logger, router }: Server$opts) {
+  constructor({ logger, router, server: { cors } }: Server$opts) {
```

Comment:
That's a good call. If we add more config, we can add the keys individual here as need once we spread the server config out in the constructor call in `./src/application/initialize.js`